src/model.js

```diff
@@ -256,6 +256,10 @@ export class Model {
           _autoGenerated: true,
         },
       };
+
+      if (Object.keys(this.primaryKeys).length === 0) {
```

Comment:
Interesting that this doesn't fail a test. Since in line 243 it mentioned that `this.primaryKeys` can't be used there. Is that outdated documentation or do we just not have proper tests for that?

Reply:
> Interesting that this doesn't fail a test. Since in line 243 it mentioned that `this.primaryKeys` can't be used there. Is that outdated documentation or do we just not have proper tests for that?

I think it was outdated documentation, and it feels like the line was for default primary key when none is defined in attributes object, so I put it there and everything goes okay.

Reply:
The comment was `we can't use here` and basically no errors now. So I think it's safe to delete this outdated documentation for now. 😃

Reply:
I can confirm that by the time `_addDefaultAttributes` is called, `this.primaryKeys` is always equal to an empty object. It's populated by `refreshAttributes` which is called after `_addDefaultAttributes`

These couple of lines do nothing in theory, `primaryKeys` will be overwritten as soon as this function ends

It also means that the comment
> Can't use this.primaryKeys here, since this function is called before PKs are identified

is right, we can't use `this.primaryKeys` in here